tests/ui-tests/tests/cases/input-fragment-no-graphql-type.stderr

```diff
@@ -0,0 +1,15 @@
+warning: unused import: `types::*`
+ --> $DIR/input-fragment-no-graphql-type.rs:7:28
+  |
+7 |     use super::{query_dsl, types::*};
+  |                            ^^^^^^^^
+  |
+  = note: `#[warn(unused_imports)]` on by default
+
+warning: unused import: `super::types::*`
+  --> $DIR/input-fragment-no-graphql-type.rs:61:9
+   |
+61 |     use super::types::*;
+   |         ^^^^^^^^^^^^^^^
+
+warning: 2 warnings emitted
```

Comment:
Would be good to fix these warnings so this case compiles clean - think you just need to get rid of the `types::*` imports and it'll be good?

Reply:
We need https://github.com/obmarg/cynic/pull/212